app/models/create_draft_edition.rb

```diff
@@ -13,6 +13,13 @@ def call
 
 private
 
+  DATE_FIELDS = {
```

Comment:
It's annoying we have this duplication between the create and the update class, I wonder if we can create a subclass or a mixin?